proxy/proxy.go

```diff
@@ -105,18 +105,44 @@ func (p *Proxy) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	// Determine auth level required to access service and dispatch request
 	// accordingly.
 	authLevel := target.AuthRequired(r)
+
+	// Determine if the header contains the necessary authentication
+	// required for the given resource.
+	resourceName := target.ResourceName(r.URL.Path)
+	acceptAuth := p.authenticator.Accept(&r.Header, resourceName)
```

Comment:
Oops, forgot to submit the comment...

I think this would now be called for every call, even static resources. This is probably too much of a performance hit as this might make one or more calls to `lnd`. We probably need to move the `Accept()` call inside the `switch` case again. Can add a comment to explain why it's in both `case` bodies.

Reply:
oh yes of course! thanks for catching that!